### ISSUE TITLE
Upvote this if this `@warn` is driving you crazy

### ISSUE THREAD

**Issue by GearsAD** [JuliaRobotics/DistributedFactorGraphs.jl#141]
```
└ @ DistributedFactorGraphs ~/.julia/dev/DistributedFactorGraphs/src/services/AbstractDFG.jl:25
┌ Warning: Retrieving serialization module from AbstractDFG - override this in the 'CloudGraphsDFG{SolverParams} structure! This is returning Main
```

**Comment by Affie** [comment]
Not yet, but I modified my code to only display it once ;-)
The same with getData, that was driving me mad:
Warning: getData is deprecated, please use solverData(), future warnings in getData is suppressed

**Comment by dehann** [comment]
Latest IIF should fix this (v0.7.9) `solverData` thing too.

**Comment by GearsAD** [comment]
Cool and the main issue is resolved (for now) in v0.5.0!
